### PR TITLE
Add djv library to the list of validators

### PR DESCRIPTION
**Pull request by korzio**
Hi there,
I've added a *djv* library which stands for *Dynamic JSON Schema Validator*. Current package supports **JSON Schema v4** and it contains utils for validating objects against schemas. It has a version 1.0.0-beta, but I will remove a suffix soon.
Thanks!

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -95,13 +96,29 @@ testRunner([
...
 		test: function (instance, json, schema) {
 			return instance.validate(schema, json) === null;
 		}
 	},
 	{
+		name: 'djv',
+		setup: function (schema) {
+			var validator = djv();
+			var schemas = Object.keys(refs);
+			schemas.splice(3, 1);
```
Comment (ebdrup) [inline index.js]: Thanks a lot for your contrbution
This code you have here for setup is against the guidelines for adding validators. The setup should just be using the validator as it comes out of the box.

Comment (korzio) [inline index.js]: @ebdrup 
So if I don't support draft03, can I still contribute?
Thanks!

**Comment by ebdrup** [comment]
Sure. If this is because of draft3. I would rather see all validators only tested against draft 4. Than doing special code for one validator only.

File: index.js [hunk 2]
```diff
@@ -106,13 +107,7 @@ testRunner([
...
 		name: 'djv',
 		setup: function (schema) {
-			var validator = djv();
-			var schemas = Object.keys(refs);
-			schemas.splice(3, 1);
-			schemas.forEach(function (uri) {
-				validator.addSchema(uri, refs[uri]);
-			});
-			return validator.addSchema('test', schema).fn;
+			return djv.addSchema('test', schema).fn;
```
Comment (korzio) [inline index.js]: @ebdrup is it what you've meant? 
I've added a static configuration on line:35 - I add necessary schemas, excluding the draft03, as it is not supported. Also now I create a single environment as some other validators do, so in a setup I just add a schema to validate object against.

Comment (epoberezkin) [inline index.js]: looks good

**Comment by epoberezkin** [comment]
@korzio +1
Now I have some motivation to improve Ajv performance :)
I have a "compliance benchmark" here: https://github.com/epoberezkin/test-validators

**Comment by korzio** [comment]
@epoberezkin 
It was a good impulse for me, thanks! However I still have to do all the job regarding v5 (and v6 proposal), that you've done a long ago.
I'll try to contribute your benchmark 👍

**Comment by epoberezkin** [comment]
@korzio I've added

**Pull request merged**